llvm/lib/DebugInfo/LogicalView/Core/LVScope.cpp

```diff
@@ -1011,9 +1011,13 @@ void LVScope::printExtra(raw_ostream &OS, bool Full) const {
   // Do not print any type or name for a lexical block.
   if (!getIsBlock()) {
     OS << " " << formattedName(getName());
-    if (!getIsAggregate())
+    if (getIsAggregate()) {
```

Comment:
Looking at the DWARF specifications, `DW_AT_byte_size` is valid for arrays, enumeration type, etc. The DWARF reader does not have any restriction while parsing the tag.
I think we should remove the `if (getIsAggregate())` condition. Just check if the value is not zero.